Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -46,14 +46,14 @@ struct DBOnboardingUX {
 class DefaultBrowserOnboardingViewController: UIViewController {
     // Public constants
     let viewModel = DefaultBrowserOnboardingViewModel()
-    static let theme = BuiltinThemeName(rawValue: ThemeManager.instance.current.name) ?? .normal
+    let theme = ThemeManager.instance.currentName
     // Private vars
     private var fxTextThemeColour: UIColor {
         // For dark theme we want to show light colours and for light we want to show dark colours
-        return DefaultBrowserOnboardingViewController.theme == .dark ? .white : .black
+        return theme == .dark ? .white : .black
     }
     private var fxBackgroundThemeColour: UIColor {
-        return DefaultBrowserOnboardingViewController.theme == .dark ? UIColor(rgb: 0x1C1C1E) : .white
+        return theme == .dark ? UIColor(rgb: 0x1C1C1E) : .white
```

Comment:
nit: We should add this color to our `FirefoxColors.swift` or `photon-colors.swift`